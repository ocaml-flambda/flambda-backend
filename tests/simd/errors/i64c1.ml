open Stdlib

type t = int64x2

external const1 : int64 -> t = "" "caml_int64x2_const1"
        [@@noalloc] [@@unboxed] [@@builtin]

let _ = const1 (Sys.opaque_identity 1L)
