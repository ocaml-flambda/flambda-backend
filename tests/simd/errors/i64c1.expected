File "i64c1.ml", line 1:
Error: Did not find constant int64 arguments for caml_int64x2_const1
